generator/config/expression/setIntersection.yaml

```diff
@@ -12,3 +12,33 @@ arguments:
         type:
             - resolvesToArray
         variadic: array
+tests:
+    -
+        name: 'Elements Array Example'
+        link: 'https://www.mongodb.com/docs/manual/reference/operator/aggregation/setIntersection/#elements-array-example'
+        pipeline:
+            -
+                $project:
+                    flowerFieldA: 1
+                    flowerFieldB: 1
+                    commonToBoth:
+                        $setIntersection:
+                            - '$flowerFieldA'
+                            - '$flowerFieldB'
+                    _id: 0
+    -
+        name: 'Retrieve Documents for Roles Granted to the Current User'
+        link: 'https://www.mongodb.com/docs/manual/reference/operator/aggregation/setIntersection/#retrieve-documents-for-roles-granted-to-the-current-user'
+        pipeline:
+            -
+                $match:
+                    $expr:
+                        $not:
+                            # the example doesn't use an array inside $not, but the documentation say it is necessary
```

Comment:
The [documentation for `$not`](https://www.mongodb.com/docs/manual/reference/operator/aggregation/not/) is clear that the expression must be in a single-value array. But the [example](https://www.mongodb.com/docs/manual/reference/operator/aggregation/setIntersection/#retrieve-documents-for-roles-granted-to-the-current-user) puts the expression directly.